src/main/java/com/aws/greengrass/clientdevices/auth/certificate/CISShadowMonitor.java

```diff
@@ -209,16 +210,23 @@ private void subscribeToShadowTopics() throws InterruptedException {
     }
 
     private void processCISShadow(GetShadowResponse response) {
-        processCISShadow(response.version, response.state.desired);
+        String cisVersion = (String) response.state.desired.get("version");
```

Comment:
blind casting is scary. Use `Coerce.toString` at least

Reply:
Is this change backwards/forwards compatible? If the customer updates to this version of CDA and then rolls back, will this change break anything?

Reply:
yep, completely missed that

Reply:
Un-resolved. Can you answer the question about compatibility too?

Reply:
sorry didn't see the compatibility comment at first--- no this won't affect compatibility,  the shadow contents itself aren't being changed